### DISCUSSION TITLE
What is the strictMode option for?

### DISCUSSION THREAD

**Discussion by Axedyson** [i18next/next-i18next#1853] [Q&A] [answered]
When looking at the [configuration options](https://github.com/isaachinman/next-i18next#options) I see there is an option called `strictMode`. But I don't see any documentation for it, what does this option do exactly when it's set to `false`?

**Comment by isaachinman** [comment, marked as answer]
Actually, I believe `strictMode` is now an unused config option. It should be removed.

https://github.com/isaachinman/next-i18next/issues/1854
